cmd/controller-manager/app/options/csrsigningcontroller.go

```diff
@@ -22,6 +22,16 @@ import (
 	"k8s.io/kubernetes/pkg/apis/componentconfig"
```

Comment:
this (and a number of other option structs) can move into the kube-controller-manager now, can't it?

Follow-up is fine (otherwise github will make a review hell out of this I fear).

Reply:
Yes, for thoes struct just for `kube-controller manager`. Will do later

Reply:
> this (and a number of other option structs) can move into the kube-controller-manager now, can't it?
> 
> Follow-up is fine (otherwise github will make a review hell out of this I fear).

I saw this and though the same thing.  But like @sttts , I think this pull moves us far enough forward to warrant merging without requiring the move here.